Delcam.ProductInterface.PowerMILL.Test/PMEntityTests/PMToolTest.cs

```diff
@@ -513,6 +513,28 @@ public void HolderOverHangTest()
             Assert.AreEqual((MM) 45.0, tool.Overhang);
         }
 
+        [Test]
+        public void HolderLengthTest()
+        {
+            if (_powerMill.Version < new Version("15.0"))
+            {
+                Assert.Inconclusive("Test not available for this version of PowerMILL");
+            }
+            var tool = _powerMill.ActiveProject.Tools[12];
+            Assert.AreEqual((MM)50.0, tool.HolderLength());
+        }
+
+        [Test]
+        public void GaugeLengthTest()
+        {
+            if (_powerMill.Version < new Version("15.0"))
+            {
+                Assert.Inconclusive("Test not available for this version of PowerMILL");
+            }
+            var tool = _powerMill.ActiveProject.Tools[12];
+            Assert.AreEqual((MM)95.0, tool.GaugeLength());
```

Comment:
Can you use Assert.That((MM)95.0, Is.EqualTo(tool.GaugeLength());  This is the preferred method of doing it.